### DISCUSSION TITLE
teamlead team timesheets himself

### DISCUSSION THREAD

**Discussion by portosricos** [kimai/kimai#3669] [Questions & Answers]
Kimai is great but I have a small question about the logic of team timesheets.

In settings:
Timetecking mode => [Time-clock]

in roles:
team lead can create other timesheet and create his own timesheet.

My question:
How can I remove the teamlead name in the user list?

thank you

**Comment by kevinpapst** [comment]
screenshot please, i do not understand

**Reply by portosricos** [reply]
Sorry can't do a screenshot.

In admin-> timesheet

A teamlead can verify the records of his team. He can add, edit or delete a entry.
When he add, he can put the name of the employee and add the start time + end time.

In the drop down menu, you can see all the employee that he has in his team. I would like to remove the user in the list of employee since  they are suppose to have the time-clock option.

example:

User A is the teamlead of User B and User C

right now, in the list of user, User A will see

User A
User B
User C

I would like 
User B 
User C

**Reply by kevinpapst** [reply]
The admin view for all users has no limitations on the edit screen. 
It is there to fix issues with wrong timesheets.

Yes, I understand your point and it is an interesting edge case that nobody ever mentioned before. 
You could deactivate edit_other_timesheets for teamleads.
But I believe that at some point you have to trust your users.
There is always some unexpected way around limitations, and if it is two teamleads changing timesheet for each other.
